app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_post_comments_counter
+
+  def update_post_comments_counter
+    post.update_comments_counter
+  end
```

Comment:
- [ ] Kindly use `after_save` because as for a database that can be updated , I think after_save it prefered because `after_create` has no effect on update 😄

- [ ] Only the methods for updating should be `private` because they are excecuted internally 😄 , Kindly update that 👍

- [ ] [OPTIONAL] I will suggest to use `increment` instead of `update` because `increment` method performs the increment operation directly on the database level, ensuring atomicity. it will increase the efficiency as you are perfomiing simple logic 👍 . Kindly check[ here](https://stackoverflow.com/questions/47790927/rails-increment-a-column-and-update-attributes) for more details